sdk/spring/azure-spring-boot-starter-keyvault-certificates/src/main/java/com/azure/spring/security/keyvault/certificates/starter/AzureCertPathProperties.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.security.keyvault.certificates.starter;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+
+/**
+ * This is used to generate spring-configuration-metadata.json
+ *
+ * @see <a href="https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-configuration-metadata.html">Metadata</a>
+ */
+@EnableConfigurationProperties({ AzureCertPathProperties.class })
+@ConfigurationProperties("azure.cert-path")
```

Comment:
we should discuss this property name, `azure.cert-path.custom` seems not a clear name to me

Reply:
Current property name is the conclusion discussed with @gavinfish
Refs: https://github.com/Azure/azure-sdk-for-java/issues/21611#issuecomment-847594035